### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by Making24** [jxhuang0508/FSDR#1]
Hi @jxhuang0508  @Dayan-Guan, thanks for sharing code for your great work.

However, I can only find the testing code. Could you also provide the source code? Thanks!

Also, could you provide the results of VGG, ResNet101 for both FSRD and IBN-Net. I would like to reproduce them.

And, could you also provide the testing code on BDD and Maphillary?

Thanks for your consideration!

**Comment by jxhuang0508** [comment]
Thanks for your attention.

I cannot guarantee the training code release as this work is with the company, where the techniques are under patent filing and the release requires their approval.

Thanks.

**Comment by Making24** [comment]
@jxhuang0508 @Dayan-Guan Thanks for your quick reply!

I understand your company policy. But can you provide the training details of your approach and IBN-NET. For example, optimizer, learning rate, training iteration, bath size, schedule policy. In addition, in Advent, they use the bs=1 and fix the bn layers, did you apply it the same?

Moreover, since IBN-NET is implemented by your paper, could you provide the training code and testing code for it? Also, the testing code for other datasets is not provided. Could you also provide the pretrained models for SYNTHIA? I think these training of IBN-NET and testing code do not violate your company policy. 

Thanks for your consideration!

**Comment by maosu1314** [comment]
Do you re-implement the work successfully? I try to re-implement it but cannot reproduce the reported performance.

**Comment by Making24** [comment]
> Do you re-implement the work successfully? I try to re-implement it but cannot reproduce the reported performance.

No, I failed to re-implement the method. But I can reproduce the results with the pre-trained model in the testing phase. @maosu1314 You can regard it as a testing approach in your paper.

**Comment by lxtGH** [comment]
@Making24 Hi! I wonder whether this paper can reproduced?

**Comment by Making24** [comment]
> 

Not yet. It is too hard for me, my bad.

**Comment by Making24** [comment]
> @Making24 Hi! I wonder whether this paper can reproduced?

@lxtGH Not yet. It is too hard for me, my bad.
